static/js/components/ThumbnailList.jsx

```diff
@@ -42,69 +79,76 @@ const Thumbnail = ({ ra, dec, telescope, mjd, name, url }) => {
       link = "";
   }
 
-  // Always apply Thumbnail style; conditionally apply DR8 style
-  const thumbnailDivClassNames = clsx(
-    styles.Thumbnail, {
-      [styles.dr8]: name === "dr8"
-    }
-  );
-
-  const thumbnailClassNames = clsx(
-    {
-      [styles.dr8crosshairs]: name === "dr8"
-    }
-  );
 
   return (
-    <a href={link}>
-      {name === "dr8" && <br />}
-      <div className={thumbnailDivClassNames}>
-        <b>
+    <Card className={classes.root} variant="outlined">
+      <CardContent>
+        <Typography className={classes.title} color="textSecondary">
           {name.toUpperCase()}
-        </b>
-        <br />
-        <div className={styles.thumbnailimgdiv}>
-          <img className={thumbnailClassNames} src={url} alt={alt} title={alt} />
-          {
-            (name === "dr8") &&
-              <img className={thumbnailClassNames} src="/static/images/crosshairs.png" alt="" />
-          }
-        </div>
+        </Typography>
+      </CardContent>
+      <div className={classes.mediaDiv}>
+        <a href={link}>
+          <img src={url} alt={alt} className={classes.media} title={alt} />
+        </a>
+        {
+          (name === "dr8") && <img className={classes.crosshair} src="/static/images/crosshairs.png" alt="" />
+        }
       </div>
-    </a>
+    </Card>
   );
 };
 
 Thumbnail.propTypes = {
   ra: PropTypes.number.isRequired,
   dec: PropTypes.number.isRequired,
-  telescope: PropTypes.string.isRequired,
   name: PropTypes.string.isRequired,
   url: PropTypes.string.isRequired,
-  mjd: PropTypes.number.isRequired
 };
 
-const ThumbnailList = ({ ra, dec, thumbnails }) => {
+const ThumbnailList = ({ ra, dec, thumbnails, useGrid=true }) => {
   const thumbnail_order = ['new', 'ref', 'sub', 'sdss', 'dr8'];
   // Sort thumbnails by order of appearance in `thumbnail_order`
   thumbnails.sort((a, b) => (thumbnail_order.indexOf(a.type) <
-                             thumbnail_order.indexOf(b.type) ? -1 : 1));
+  thumbnail_order.indexOf(b.type) ? -1 : 1));
 
-  return (
-    <div className={styles.ThumbnailList}>
-      {thumbnails.map((t) => (
-        <Thumbnail
-          key={`thumb_${t.type}`}
-          ra={ra}
-          dec={dec}
-          name={t.type}
-          url={t.public_url}
-          telescope={t.photometry.instrument.telescope.nickname}
-          mjd={t.photometry.mjd}
-        />
-      ))}
-    </div>
-  );
+  if (useGrid) {
```

Comment:
Are these changes supposed to be in here? Are they needed for the observing planner component?

Reply:
Yeah.